src/main/java/io/jenkins/plugins/services/impl/HttpClientJiraIssues.java

```diff
@@ -60,18 +61,20 @@ public JiraIssues getIssues(String pluginName, int startAt) throws IOException {
     JiraIssues jiraIssues = new JiraIssues();
 
     String query = URLEncoder.encode("project=JENKINS AND status in (Open, \"In Progress\", Reopened) AND component=" + component, "UTF-8");
-    String jsonInput = getHttpContent("/rest/api/2/search?startAt=" + startAt + "&maxResults=" + maxResults + "&jql=" + query, Collections.emptyList());
+    String url = "/rest/api/2/search?startAt=" + startAt + "&maxResults=" + maxResults + "&jql=" + query;
+    String jsonInput = getHttpContent(url, Collections.emptyList());
     if (Strings.isNullOrEmpty(jsonInput)) {
-      throw new IOException("Empty return value");
+      String msg = "[" + pluginName + "] Empty return value for " + url;
+      logger.debug(msg);
+      Sentry.capture(new Error(msg));
+      return jiraIssues;
     }
 
     JSONObject obj = new JSONObject(jsonInput);
     if (obj.has("errorMessages")) {
-      if (obj.getJSONArray("errorMessages").join("|").contains("\"The value '" + component + "' does not exist for the field 'component'.\"")) {
-        logger.debug("JSON Response with error: " + jsonInput);
-        return jiraIssues;
-      }
-      throw new IOException(jsonInput);
+      logger.debug("[" + pluginName + "] JSON Response with error: " + jsonInput);
```

Comment:
any reason this isn't at warning?
```suggestion
      logger.warn("[" + pluginName + "] JSON Response with error: " + jsonInput);
```